gitclone/gitclone_test.go

```diff
@@ -12,608 +12,565 @@ import (
 
 const rawCmdError = "dummy_cmd_error"
 
-var checkoutStateTestCases = [...]struct {
```

Comment:
I just moved this list inside the function body so that GoLand recognizes each test case individually and can navigate to the failing test case. Sorry for the messy diff :(

Reply:
There are a few changes though in a few test cases that were incorrect